### PR TITLE
Remove touchstart even on TouchSpin on FO

### LINKED ISSUE #23989: Accidentally change product order quantity when swipe on touchspin
When using Prestashop on smartphone/tablet or any other device with touchscreen.
If while you're on cart page you scroll the page swiping finger on the touchspin "+" and "-" the product quantity will change, like if you clicked on it.
This can cause to accidentally change product quantity.

I think that "+" and "-" button should increase (or decrease) product quantity only when you click on it, and not just swiping on it.
In particular on smarphone, because the position it's very close to where you normally swipe with thumb when using it with one hand.

1. Go to front office using a device with touchscreen display
2. Add anything to the cart
3. Open cart
4. scroll up and down swiping finger on the touchspin "+" and "-"

#### Additional information

* PrestaShop version: 1.7.7.3
* PHP version: 7.3

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Touchspin is boring when you start scrolling on it on mobile, it should not work if you drag on it from mobile
| Type?             | bug fix
| Category?         | FO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #23989.
| How to test?      | Go on product page, listing and cart, go on mobile mode and try to scroll by beginning on the touchspin
| Possible impacts? | Mobile use on FO

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24012)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

File: themes/classic/_dev/js/product.js [hunk 1]
```diff
@@ -137,8 +137,11 @@ $(document).ready(() => {
       buttonup_class: 'btn btn-touchspin js-touchspin',
       min: parseInt($quantityInput.attr('min'), 10),
       max: 1000000,
+      mousewheel: false,
```
Comment (atomiix) [inline themes/classic/_dev/js/product.js]: Is it disabled only here on purpose?

Comment (NeOMakinG) [reply themes/classic/_dev/js/product.js]: You were right, I added it while trying some things, but it's not needed

**Comment by NeOMakinG** [comment]
@Progi1984 I used the selectors map and removed a wrong conf

**Review by Progi1984 — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @NeOMakinG and @sarahdib

**Pull request merged**